config/initializers/rack_timeout.rb

```diff
@@ -1 +1,2 @@
+Rack::Timeout::Logger.disable if Rails.env.development?
```

Comment:
I would leave this as is. 
Production-development parity is important, breaking updates on rack-timeout should be caught on development.